src/libumf.map.in

```diff
@@ -55,6 +55,7 @@ UMF_1.0 {
         umfProxyPoolOps;
         umfPutIPCHandle;
         umfScalablePoolOps;
+        umfDevDaxMemoryProviderOps;
```

Comment:
so since this is added only under `UMF_DISABLE_HWLOC` this symbol should also be added via `add_optional_symbol` (similarly to `umfOsMemoryProviderOps`) - see top-level CMake

Reply:
Done